include/library/spdm_lib_config.h

```diff
@@ -59,6 +59,18 @@
 #define LIBSPDM_MAX_CONNECTION_STATE_CALLBACK_NUM 4
 #endif
 
+#ifndef LIBSPDM_MAX_REQUSET_INFO_SIZE
+#define LIBSPDM_MAX_REQUSET_INFO_SIZE 0x300
+#endif
+
+#ifndef LIBSPDM_MAX_CSR_SIZE
+#define LIBSPDM_MAX_CSR_SIZE 0x1000
+#endif
+
+#ifndef LIBSPDM_MAX_PRIVATE_KEY_SIZE
+#define LIBSPDM_MAX_PRIVATE_KEY_SIZE 0x1000
```

Comment:
Why we need this MACRO?

Reply:
I have changed the private key array to pointer.